content/en/docs/event-sources/kernel/architecture.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Kernel Events Architecture
+description: 'Architecture overview for kernel events.'
+linktitle: Kernel Events Architecture Overview
+weight: 20
+---
+
+This document describes the overall architecture that allows events from kernel sources to be ingested by Falco and how to use the libraries to inspect the data collection flow, and how Falco manages the boundary between the kernel and userspace.
```

Comment:
I like this, but we should clarify that these APIs aren't intended for adopters.

These APIs only allow Falco to employ the drivers, so they must be considered for internal use only and are subject to change at any time.